Bio/Nexus/Nexus.py

```diff
@@ -2133,7 +2133,7 @@ def terminal_gap_to_missing(self, missing=None, skip_n=True):
 
 
 try:
-    import cnexus
+    import cnexus  # type: ignore
```

Comment:
This import actually looks wrong. The module name is `Bio.Nexus.cnexus`. Broken since Python 3 migration?

Reply:
Fix: https://github.com/biopython/biopython/pull/4413

Reply:
I think you're probably right about that - good catch.